src/__experimental__/components/numeral-date/numeral-date.component.js

```diff
@@ -0,0 +1,140 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import invariant from 'invariant';
+import Events from '../../../utils/helpers/events';
+import StyledNumeralDate, { StyledDateField } from './numeral-date.style';
+import Textbox from '../textbox';
+
+const NumeralDate = ({
+  dateFormat,
+  defaultValue,
+  errorMessage,
+  errorPresent,
+  id,
+  name,
+  onBlur,
+  onChange,
+  value
+}) => {
+  const isControlled = value !== undefined;
+  const initialValue = isControlled ? value : defaultValue;
+  invariant(
+    initialValue !== undefined,
+    'This component has no initial value'
+  );
+
+  let inputRef = useRef();
+  const [isActive, setIsActive] = useState(inputRef.current === document.activeElement);
```

Comment:
Is this check  `inputRef.current === document.activeElement` needed here? When component is initialized the inputRef.current is always `undefined` on the first run so the `isActive` state value will default to false anyway.